docs/zh/latest/how-to-build.md

```diff
@@ -115,7 +115,18 @@ sudo yum install ./apisix/*.rpm
 
 - 4.1 `make deps` 安装 `lualdap` 失败, 错误信息如: `Could not find header file for LDAP`
 
-      解决方案: 通过 `luarocks config` 手动设置 `LDAP_DIR` 变量, 比如 `luarocks config variables.LDAP_DIR /usr/local/opt/openldap/`
+      解决方案: luarocks 支持添加自定义的包地址到项目中。通过 `luarocks config` 手动设置 `LDAP_DIR` 变量, 比如 `luarocks config variables.LDAP_DIR /usr/local/opt/openldap/`，你应该按照下面流程设置：
+      1. 使用包括但不仅限于 `brew` 等工具将 `openldap` 安装到本地;
+      2. 找到 `openldap` 的本地安装目录。如果是 macOS 系统，并使用 `brew` 安装 `openldap` 的话，可以通过 `brew --prefix openldap` 命令找到本地安装目录。
```

Comment:
can you add and supplement the installation and search methods of `Ubuntu` and `CentOS` systems?
changes in the `how-to-build` documentation need cover as many mainstream operating systems as possible.

Reply:
OK, I will try.